README.md

```diff
@@ -35,9 +35,17 @@ make run
 
 2. When connections are established, you can access:
 
-    * the LMS server on [localhost:8072](http://localhost:8072)
+    * the LMS server on [localhost:8072/login](http://localhost:8072/login)
+    * the CMS server on [localhost:8082/signin](http://localhost:8072/signin)
     * the Graylog server on [localhost:9000](http://localhost:9000)
```

Comment:
Apropos, I'm not able to access graylog server at `localhost:9000` - getting `CORS Missing allow origin` errors. However `127.0.0.1:9000` worked for me. Is it the same for you or rather an edge case for my setup ?

Reply:
Usually, the localhost is 127.0.0.1? Is it configured like this on your computer? in `/etc/hosts`

Reply:
Yes, but I think it's my `firefox` complaining that `localhost:9000` is making post requests to `127.0.0.1:9000` without `allow origin` headers. Updated `GRAYLOG_HTTP_EXTERNAL_URI` to point to `localhost`) I hope this could work for both of us)

Reply:
You are right, that's better to directly point to localhost in GRAYLOG_HTTP_EXTERNAL_URI. Thanks for the proposition!

Reply:
It works for me too!  :sparkles: